### DISCUSSION TITLE
Pocketbase adapter

### DISCUSSION THREAD

**Discussion by Bemajster** [nextauthjs/next-auth#5834] [Ideas]
### Description 📓

Add pocketbase [https://pocketbase.io] adapter to next-auth. Pocketbas eis a new NOSQL database. It looks preety promising.

### How to reproduce ☕️

_bruh_

### Contributing 🙌🏽

Yes, I am willing to help implement this feature in a PR

**Comment by deepak-coding-art** [comment]
Did you find anything

**Comment by tajkirkpatrick** [comment]
If anyone happens to stumble on this in the future, I'd appreciate any help in [this discussion.](https://github.com/pocketbase/pocketbase/discussions/1669)

**Comment by 0xsarwagya** [comment]
Our Team Has Created An Adapter Recently Since We Are Using Pocketbase Now.

It's Still Under Active Development and Contributions are Welcome

[You Can look at The Github Repo Here](https://github.com/Rebackk-Team/pocketbase-adapter)
[Also The NPM Package Here](https://www.npmjs.com/package/@rebackk/pocketbase-adapter)
